bokehjs/src/coffee/renderer/annotation/box_annotation.coffee

```diff
@@ -48,14 +82,25 @@ class BoxAnnotation extends HasParent
   default_view: BoxAnnotationView
   type: 'BoxAnnotation'
 
+  nonserializable_attribute_names: () ->
+    super().concat(['silent_update'])
+
+  update:({left, right, top, bottom}) ->
+    if @get('silent_update')
+      @attributes['left'] = left
+      @attributes['right'] = right
+      @attributes['top'] = top
+      @attributes['bottom'] = bottom
```

Comment:
something needs investigating here... the patch events are from the change notification, and silent mode should suppress change notification... not sure the patch events you saw were caused by this? must be something going on